lms/djangoapps/instructor/views/api.py

```diff
@@ -2090,6 +2090,30 @@ def get_ora2_responses(request, course_id):
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @require_level('staff')
+def get_course_forums_usage(request, course_id):
+    """
+    Pushes a Celery task which will aggregate course forum statistics into a .csv
+    """
+    course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
+    try:
+        instructor_task.api.submit_course_forums_usage_task(request, course_key)
+        success_status = _("The course forum usage statistics is being generated.")
```

Comment:
Is it confusing to stay the statistics are being generated rather than the report? If the instructors don't care, might just be simpler to say "The course forums usage report is being generated". Also should it be course forum (singular) or course forums (plural)?